content/writing guide.markdown

```diff
@@ -0,0 +1,58 @@
+**Support Writing Guidelines** 
+
+**Tone**
+
+* Straightforward and friendly. 
+* We want to be professional, and convey knowledge, but without alienating new users or those who aren't 100% familiar with the terms.
+
+**Use gender-neutral language**
+
+* Always opt for they/them/you language over she/him.
+* They and their can be used as singular pronouns.  
+* Don't default to masculine pronouns. 
+
+**Tense**
+
+* Always use present tense when applicable. It's stronger.
+* Every sub-header (when possible) should use the present continuous form. 
```

Comment:
Out of curiosity, I've been reading several handbook and I found that a more direct form is generally used. For instance, what about:

> Use the present continuous form in every sub-header, when possible: 

vs

> Every sub-header (when possible) should use the present continuous form.